README.md

```diff
@@ -85,393 +51,9 @@ JwksEndpoint::fromRegistration($registration)->outputJwks();
 JwksEndpoint::new(['a_unique_KID' => file_get_contents('/path/to/private/key.pem')])->outputJwks();
 ```
 
-## Handling Requests
```

Comment:
Moved here: https://github.com/packbackbooks/lti-1-3-php-library/wiki/Endpoints-and-Handling-Requests